templates/taxi/car_detail.html

```diff
@@ -12,7 +12,20 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    Drivers
+
+    {% if car in user.cars.all %}
```

Comment:
The condition `{% if car in user.cars.all %}` is incorrect. It should check if the user is assigned to the car, not if the car is in the user's cars. Consider using `{% if user in car.drivers.all %}` to check if the current user is a driver of the car.